server/Pipfile

```diff
@@ -1,19 +0,0 @@
-[[source]]
```

Comment:
Pipfile shouldn't be deleted

Reply:
Added pipfile back to PR

Reply:
I don't know the repercussions of rolling back to the commit where it still existed, will ask in standup tomorrow